packages/main/src/FileUploader.hbs

```diff
@@ -6,6 +6,9 @@
 	@keydown="{{_onkeydown}}"
 	@keyup="{{_onkeyup}}"
 	@click="{{_onclick}}"
+	@dragover="{{_ondrag}}"
+	@dragenter="{{_ondrag}}"
```

Comment:
What's the default behaviour that we want to prevent and stop propagation of? For drag over I understand that we need to prevent the default behaviour so we can have the drop, but why are we doing it for dragenter?

Reply:
Turns out it is not necessary to listen and prevent on `dragenter` event. Was mimic a similar solution from openui5 project.